src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarMain.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import ChromosomeNavigator from 'src/content/app/browser/chromosome-navigator/ChromosomeNavigator';
+import BrowserNavBarRegionSwitcher from './BrowserNavBarRegionSwitcher';
+import { ReactComponent as CloseIcon } from 'static/img/shared/close.svg';
+
+import styles from './BrowserNavBarMain.scss';
+
+export enum Content {
+  CHROMOSOME,
+  REGION_SWITCHER
+}
+
+const BrowserNavBarMain = () => {
+  const [view, changeView] = useState<Content>(Content.CHROMOSOME);
+
+  const onChangeViewClick = (newView: Content) => {
```

Comment:
Can this be renamed to something like `onSwitchHandler`? 
Having `Click` after `View` sounds a bit strange.

Reply:
I have never seen this written down in an influential style guide, but there are two traditions in naming of event handlers.

According to one tradition, they are named with the `on` prefix followed by the name of the event — `onClick`, `onScroll`, `onMouseEnter`, etc. This is how React names its own events, which map to the DOM event names (`onclick`, `onscroll`, `onmouseenter`, etc.). Here's someone [thinking out loud](https://jaketrent.com/post/naming-event-handlers-react/) in their blog the way they name events in a similar fashion.

According to the other tradition, they are named with the `handler` suffix, following the event name (e.g. `clickHandler`, `scrollHandler`, `submitHandler`, etc.). Here's an [old Microsoft style guide](https://docs.microsoft.com/en-us/dynamicsax-2012/developer/naming-conventions-delegates-and-event-handlers#event-handler-naming-conventions) for god knows what language that exemplifies this approach.

So I try to name my events according to either of these traditions (I think both are fine). What I emphatically **do not** want us to do is to mix them in a way that combines both the `on` prefix and the `handler` suffix (i.e. `onSwitch` is ok, `switchHandler` is ok, but `onSwitchHandler` is not).

Reply:
changed `onChangeViewClick` to `handleViewChange` (`view` being the name of the variable stored in the component state and referring to whether we are showing the chromosome visualisation or the region switcher).

Reply:
I agree that it is good to follow either of these traditions but my only concern was the link between the words.
For example, `mouseEnter` can be linked as `mouse -> enter`. `changeview` can be linked as `Change` -> `view`. But having `changeViewClick` felt a bit unclear whether it means `changeView -> click` or `change -> viewClick` .